tests/test_cases/test_3316_a/Makefile

```
@@ -13,9 +13,9 @@ all:
 else ifeq ($(filter verilator,$(shell echo $(SIM) | tr A-Z a-z)),)
 all:
 	@echo "Skipping test since only Verilator is supported"
-else ifeq ($(filter $(EXTRA_ARGS),--no-timing --timing),)
-all:
-	@echo "Skipping test since Verilator requires --timing or --no-timing option."
 else
+.PHONY: override_for_this_test
+override_for_this_test:
+	if $(MAKE) all; then echo "Expected test to fail without --timing flag"; false; else echo "Failed as expected"; fi
```

```suggestion
	# we cannot normally rely on the simulator return code, but this test is specific to Verilator
	if $(MAKE) all; then echo "Expected test to fail without --timing flag"; false; else echo "Failed as expected"; fi
```
(if it's true that Verilator behaves in this way)